README.md

```diff
@@ -60,12 +60,12 @@ After making changes to the website, one should run a local instance of the proj
 1. Run browser tests mentioned above on localhost.
 2. Push changes to your GitHub Fork of the repository.
 3. Run tests on mobile devices when `your-account-name.github.io` has been updated.
-4. Replace the CNAME file you deleted when you first pulled the repository; the file should be named CNAME and have in it `standardhealthrecord.org`.
+4. Replace the CNAME file you deleted when you first pulled the repository; the file should be named CNAME and have in it `standardhealth.github.io`.
```

Comment:
There is no CNAME file anymore, so do we even need this step?

Reply:
sheesh you are so right

Reply:
```suggestion
```